internal/writer/writer_test.go

```diff
@@ -19,12 +19,6 @@ func TestWriter_Write(t *testing.T) {
 package mytestpackage
 
 // Common Code Here
-`,
-		Constants: `// Code generated by reinforcer, DO NOT EDIT.
```

Comment:
Confused about removing this. Are we not writing this "Code generated by reinforcer" comment in all files anymore?

Reply:
This PR stops generating the Constants file and instead puts each target's constants at the top of the file specific to that target (ie if you generate for `TradeServiceClient` the constants are at the top of `trade_service_client.go`)

I have no idea why they were put into a common file to begin with. It is not deterministic and if you run reinforcer twice in the same directory it OVERWRITES THIS FILE.

Reply:
If you look closely you'll see that those constants are being moved to another output file.